CONFIGURATION.md

```diff
@@ -26,11 +26,11 @@ sudo ./target/release/ntp-daemon -p pool.ntp.org
 
 After a few minutes you should start to see messages indicating the offset of your machine from the server.
 
-### New clock algorithm
+### RFC5905 compliant algorithm
 
-NTPD-rs now also supports a new, better performing clock algorithm. This algorithm doesn't conform to the specification in RFC5905, but offers significantly better performance. Experience with both it and Chrony's non-standard algorithm indicates that a different clock algorithm does not impede interoperability.
+By default, NTPD-rs uses a newer, better performing clock algorithm. This algorithm doesn't conform to the specification in RFC5905, but offers significantly better performance. Experience with both it and Chrony's non-standard algorithm indicates that a different clock algorithm does not impede interoperability.
 
-Currently, the new algorithm is opt-in, and must be selected at build time. To build it, add the additional flags `--features new-algorithm` to the build command. Please note that when using the new clock algorithm, a different set of configuration options needs to be used to tune it, as indicated below.
+However, should you desire to run an ntp instance that is to the largest extent possible RFC5905 compliant, it is possible to compile NTPD-rs with the clock algorithm from RFC5905. To do this, at build time the additional flags `--features rfc-algorithm` need to b added to the build command. Please note that when using the RFC's clock algorithm, a different set of configuration options need to be used to tune it, as indicated below.
```

Comment:
Typo: need to b**e** added to the build command.

Reply:
fixed